packages/suite/src/actions/suite/storageActions.ts

```diff
@@ -52,6 +55,42 @@ export const removeAccountDraft = async (account: Account) => {
 
 // send form drafts end
 
+export const saveFormDraft = async (key: string, draft: FormDraft) => {
+    if (!(await isDBAccessible())) return;
+    return db.addItem('formDrafts', draft, key, true);
+};
+
+export const removeFormDraft = async (key: string) => {
+    if (!(await isDBAccessible())) return;
+    return db.removeItemByPK('formDrafts', key);
+};
+
+export const saveAccountFormDrafts = (accountKey: string) => async (
+    _: Dispatch,
+    getState: GetState,
+) => {
+    if (!(await isDBAccessible())) return;
+
+    const { formDrafts } = getState().wallet;
+
+    await Promise.all(
```

Comment:
this should return only promises, without await (like saveAccountDraft)
Promise.all is performed in line 123

Reply:
Similar to the previous comment.